File: altbash/altbash.c
#include <stdio.h>
#include <stdio.h>
#include <string.h>

int main(){
    char plaintext[1024];
    printf("Insira o texto a ser codificado: ");
    scanf("%s", &plaintext);
    int a = strlen(plaintext);
    char K[1024] = "";
    int i = 0;
    for(i;i<a;i++){
        int b = plaintext[i];
        int shift = b%97;
        int newInt = 122-shift;
        char C = newInt;
        K[i] += C;
        printf("%c", K[i]);
    }
}

File: CipherAlgorithms/altbash/altbash.java
import java.util.Scanner;
public class altbash{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        System.out.println("Insira o texto a ser codificado: ");
        String texto = scan.nextLine();
        int a = texto.length();
        String cypher = "\0";
        for(int i=0;i<a;i++){
            int b = texto.charAt(i);
            int shift = b%97;
            int newInt = 122-shift;
            cypher += (char)newInt;
        }
        System.out.println(cypher);
    }
}